cmake_modules/FindCBLAS.cmake

```diff
@@ -30,31 +30,5 @@ if(NOT CBLAS_FOUND)
         CBLAS_LIBRARIES 
         CBLAS_INCLUDE_DIR)
 
-    if (APPLE)
```

Comment:
I think what may have happened was that at one time, additional libraries needed to be linked into the cblas library manually.  That is no longer the case (macOS Mojave supplies the libraries in a framework).  This code should be put back in, but active only for older versions of macOS.  CMAKE_SYSTEM_VERSION can probably be leveraged.

I will investigate.

Reply:
I've re-added this code but changed the IF() slightly so that it is not executed for macOS Mojave and later (cmake system version 18.2.0 or later).  This will be in my next commit.